### PR TITLE
Improve docker info against 1.7

### PR DESCRIPTION
**Pull request by vieux**

### CODE REVIEW

**Comment by vieux** [comment]
ping @aluzzardi @abronan i'd like to include this one in the rc2

File: api/handlers.go [hunk 1]
```diff
@@ -44,16 +44,9 @@ func getInfo(c *context, w http.ResponseWriter, r *http.Request) {
...
-		APIVersion string `json:"ApiVersion"`
-		GoVersion  string
-		GitCommit  string
-		Os         string
-		Arch       string
-	}{
+	version := dockerclient.Version{
 		Version:    "swarm/" + version.VERSION,
-		APIVersion: APIVERSION,
+		ApiVersion: APIVERSION,
```
Comment (aluzzardi) [inline api/handlers.go]: Are you sure about this one? I don't remember which one we should use

File: cluster/cluster.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ type Cluster interface {
 
 	// Return some info about the cluster, like nb or containers / images
 	// It is pretty open, so the implementation decides what to return.
-	Info() [][2]string
+	Info() ([][]string, int64, int64)
```
Comment (aluzzardi) [inline cluster/cluster.go]: Nit (feel totally free to ignore it):

It might be cleaner to have `cluster.Memory()`

WDYT?

Comment (vieux) [inline cluster/cluster.go]: I agree, and `cluster.CPUs()` it forces us to add 2 new method to the interface.

File: api/handlers.go (hunk 1 above)
Comment (vieux) [inline api/handlers.go]: This one is good.

**Comment by vieux** [comment]
@aluzzardi PTAL

**Comment by aluzzardi** [comment]
LGTM

**Pull request merged**